src/services/firebase-client/rtdb/index.ts

```diff
@@ -0,0 +1,49 @@
+import firebase from "firebase/app"
+import "firebase/database"
+import { configs as firebaseConfigs } from '../../../constants/firebase'
+
+const isFirebaseInitialized = (): boolean => !!firebase.apps.length
+
+if (!isFirebaseInitialized()) {
+  firebase.initializeApp(firebaseConfigs)
+}
+
+const db = firebase.database()
+
+
+export const create = async (key: string, value: any) => {
+  const error = await db.ref(key).set(value)
+
+  if (error) {
+    console.error('Data could not be saved')
+    console.error(error)
+  }
+}
+
+export const append = (key: string, value: any) => {
```

Comment:
Is this just syntax sugar? dont think its necessary

Reply:
this is for abstract of `firebase.db`, so a db should provide an `append` function without exposing the implementation details

Reply:
Why a db should provide `append`?
Is this a common practice? or concept? or is it your design?
How are you going to extend you "abstraction" to more db operation? wrapping over every methods that firebase provides? how about if you switch to a restful API base backend? Would your abstraction capable to migrate to that easily?

If there is no plan to move to new storage, dont encapsulate it. Otherwise, you are just over engineering. 
There are situationsIt that is okay to wrap a certain service if you are adding values, such as
- retry mechanism
- custom error handling
- blahblahblha